extensions/import/src/common/constants.ts

```diff
@@ -58,6 +58,6 @@ export const page2NameText = localize('flatFileImport.page2Name', "Preview Data"
 export const page3NameText = localize('flatFileImport.page3Name', "Modify Columns");
 export const page4NameText = localize('flatFileImport.page4Name', "Summary");
 export const importNewFileText = localize('flatFileImport.importNewFile', "Import new file");
-
+export const createDerivedColumn = localize('flatFileImport.createDerivedColumns', "Create derived column");
```

Comment:
should this be "columns"?

Reply:
No, the button is used to create one column at a time. Updating the first argument to 'flatFileImport.createDerivedColumn' for consistency.